public/doc/openapi-3-v1.0.json

```diff
@@ -3382,6 +3382,16 @@
             "readOnly": true,
             "type": "string"
           },
+          "previous_sha": {
+            "readOnly": true,
+            "title": "SHA of previous uploaded files",
+            "type": "string"
+          },
+          "previous_size": {
+            "readOnly": true,
+            "title": "Size of previous uploaded files",
+            "type": "integer"
+          },
```

Comment:
@hsong-rh Should we expose the status values and state as enum in the openapi.json

Reply:
Added